src/db/clients/postgresql.js

```diff
@@ -23,31 +23,33 @@ export default function(server, database) {
     const client = new Client(dbConfig);
 
     debug('connecting');
-    client.connect(err => {
+    client.connect(async err => {
       if (err) {
         client.end();
         return reject(err);
       }
 
       debug('connected');
+      const defaultSchema = await getSchema(client);
+
       resolve({
         wrapIdentifier,
         disconnect: () => disconnect(client),
-        listTables: () => listTables(client),
-        listViews: () => listViews(client),
-        listRoutines: () => listRoutines(client),
-        listTableColumns: (db, table) => listTableColumns(client, db, table),
-        listTableTriggers: (table) => listTableTriggers(client, table),
+        listTables: (_, schema = defaultSchema) => listTables(client, schema),
```

Comment:
hi, thanks for the review, 
just to be sure about what you say, 
you want me to me to have smth like : 
```
        listTables: (client, schema = defaultSchema) => listTables(client, schema), 
```
?

btw, the _ is a convention for unused parameter or variable ... so i think it explain why eslint does not complain